### PR TITLE
Multi sig wallet

### PR DESCRIPTION
**Pull request by miksut**
Hi guys,
I completed the SC. It now provides functionality to replace beneficiaries and validators. As discussed, these functions require replacing the entire validators/beneficiaries array in order to not mess up and to keep it simpler. Further, each beneficiary needs to claim its own share after the death has finally be confirmed (-> for security reasons). I further added some arguments in the file "Solidity_testing" which can be used to test the SC in Remix. I did some testing and did not run into an error. However, I think it would make sense to test the SC thoroughly at a later stage to be sure to not miss any edge case.

### CODE REVIEW

**Comment by julwil** [comment]
@miksut Looks very nice!

File: contracts/DeathNote.sol [hunk 1]
```diff
@@ -1,111 +1,241 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.4.21 <0.7.0;
+pragma solidity ^0.6.0;
```
Comment (julwil) [inline contracts/DeathNote.sol]: I tried to compile it in our dev environment and I got some errors because of the solidity compiler.
When I changed it back to `pragma solidity >=0.4.21 <0.7.0;` it worked

**Review by julwil — CHANGES_REQUESTED** [review]

File: contracts/DeathNote.sol (hunk 1 above)
Comment (miksut) [reply contracts/DeathNote.sol]: Done, thnx 👍

**Comment by kebohn** [comment]
@miksut gseht nice us, danke!

**Review by kebohn — APPROVED** [review]
